Libraries/Network/__tests__/FormData-test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+jest.unmock('FormData');
+
+const FormData = require('FormData');
+
+describe('FormData', function() {
+  var formData;
+
+  beforeEach(() => {
+    formData = new FormData();
+  });
+
+  afterEach(() => {
+    formData = null;
+  });
+
+  it('should return non blob null', function() {
+    formData.append('null', null);
+
+    let parts = formData.getParts();
+    expect(parts[0].string).toBe('null');
```

Comment:
Can you use toMatchObject here and below?

https://facebook.github.io/jest/docs/expect.html#tomatchobjectobject